pyproject.toml

```diff
@@ -46,6 +46,10 @@ show_missing = true
 show_contexts = true
 
 [tool.pytest.ini_options]
+filterwarnings = [
+  "error",
```

Comment:
Would you consider putting this into a separate commit to make it clear that it represents a policy change?

Reply:
Yes, happy to do that.

I'm having trouble rebasing my change, so I've moved the commits on a new branch and will open a new PR. I'll make the change there.